pages/api/api.js

```diff
@@ -0,0 +1,14 @@
+import Axios from "axios";

```

Comment:
Please do not add non-page components in the `pages` folder. Move this in components instead